packages/client/tests/functional/queryRaw-send-type-hints/tests.ts

```diff
@@ -15,7 +15,7 @@ testMatrix.setupTestSuite((suiteConfig) => {
 
     const record = await prisma.entry.findUnique({
       where: {
-        id: 1,
```

Comment:
this needs to be updated in the raw SQL query too then (here and in everywhere else in the file)

Reply:
but I'm not really sure if it's worth changing the schema in this test to use `idForProvider` in the first place since it's inherently SQL-only and it's not possible to make this test run with MongoDB

Reply:
sure, I will remove it!